tests/py/dynamo/automatic_plugin/test_flashinfer_rmsnorm.py

```diff
@@ -0,0 +1,50 @@
+import flashinfer
+import torch
+import torch.nn as nn
+import torch_tensorrt
+from parameterized import parameterized
+from torch.testing._internal.common_utils import run_tests
+from torch_tensorrt._enums import dtype
+
+from ..conversion.harness import DispatchTestCase
+
+
+@torch.library.custom_op("flashinfer::rmsnorm", mutates_args=())  # type: ignore[misc]
+def flashinfer_rmsnorm(
+    input: torch.Tensor, weight: torch.Tensor, eps: float = 1e-6
+) -> torch.Tensor:
+    return flashinfer.norm.rmsnorm(input, weight)
```

Comment:
we don't need this registration since you can directly use torch.ops.flashinfer.rmsnorm

Reply:
> we don't need this registration since you can directly use torch.ops.flashinfer.rmsnorm

Actually, I cannot directly use it since the torch custom op registration is commented out [here](https://github.com/flashinfer-ai/flashinfer/blob/f5dec3d502634cf350a88fc50cbb4d048bda16b1/flashinfer/norm.py#L85-L89)

Reply:
commented out here: https://github.com/flashinfer-ai/flashinfer/blob/f5dec3d502634cf350a88fc50cbb4d048bda16b1/flashinfer/utils.py#L242C1-L248C12